pkg/apis/v2/dingtalkconfig_types.go

```diff
@@ -20,36 +20,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Configuration of ChatBot
-type DingTalkChatBot struct {
-	// The webhook of ChatBot which the message will send to.
-	Webhook *SecretKeySelector `json:"webhook"`
-
-	// Custom keywords of ChatBot
-	Keywords []string `json:"keywords,omitempty"`
-
-	// Secret of ChatBot, you can get it after enabled Additional Signature of ChatBot.
-	Secret *SecretKeySelector `json:"secret,omitempty"`
-}
-
-// Configuration of conversation
-type DingTalkConversation struct {
```

Comment:
we still need something like DingTalkConversationConfig here to be more clear it's conversation configs